app/presenters/licences/set-up.presenter.js

```diff
@@ -28,17 +29,25 @@ const agreementDescriptions = {
  * @param {module:ChargeVersionModel[]} chargeVersions - All charge versions records for the licence
  * @param {module:WorkflowModel[]} workflows - All in-progress workflow records for the licence
  * @param {module:LicenceAgreements[]} agreements - All agreements records for the licence
+ * @param {module:ReturnVersionModel[]} returnVersions - All returns requirement records for the licence
  * @param {Object} auth - The auth object taken from `request.auth` containing user details
  * @param {Object} commonData - Licence data already formatted for the view's shared elements
  *
  * @returns {Object} The data formatted for the view template
  */
-function go (chargeVersions, workflows, agreements, auth, commonData) {
+function go (chargeVersions, workflows, agreements, returnVersions, auth, commonData) {
   return {
+    links: {
+      chargeInformation: _chargeInformationLinks(auth, commonData),
+      agreements: _agreementLinks(auth, commonData),
+      returnsRequirements: {
+        returnsRequired: `/system/licences/${commonData.licenceId}/returns-required`,
```

Comment:
This links in the old system go to here but end up redirected to ‘Select the start date for the requirements for returns’. 

But the ticket just says to get to this page.